### ISSUE TITLE
Song History

### ISSUE THREAD

**Issue by iKeiko** [LISTEN-moe/feedback#24]
This is a feature I'd really appreciate.
A legit song history would be nice instead of only the last 2-3 songs. Official radios even have a feature like that where you can just type in the time when you listened to the radio and it will show what songs played at that time. A normal song history in a separate tab would be perfectly fine too (eg. not on the frontpage).
We already have a listen.moe NP bot on twitter, so why can't we just have it "officially" and on the website? 

https://twitter.com/listen_moe_np?lang=en (NP Bot)

**Comment by bitnimble** [comment]
Although I'm not Kana, my take on it is that we've simply never needed it. You're pretty much the first person to ask for this, as not many people exactly wonder, "Hey, I wonder what that song that played at 5:04pm was". 
Seemingly, the vast majority of users are fine just checking the last few songs that played. 

Keep in mind that this site is currently entirely a voluntary effort by just a few developers, so any new features requested compete against current features on the backlog and bugfixes (which we have a lot of). We have day jobs, and some of us have families that we have to spend time with, so we can't spend a lot of time working on features that would only satisfy just a few users.

**Comment by iKeiko** [comment]
As someone who is pretty active in the discord chat I see a lot of people asking something like "I've heard a song 30min ago with <insert lyrics they can remember>, can someone tell me what song it was?".

This isn't really something nitpicky either, since it's a music streaming website essentially. It would be nice to know what songs you listened to while you did something and couldn't check the radio/favorite the song. Of course you could just check the NP Bot but most people don't know about it and it would be a great QoL change.

**Comment by bitnimble** [comment]
Ah yeah, I'm not saying that the feature request is rejected, just trying to explain why we haven't really made an effort to put one in yet (especially because the twitter exists in the first place, funnily enough). Whether or not it's added into the actual site depends on whether Kana wants it to happen and if we get time with all the v4 bugfixes going around 🙂 

TLDR response to "why can't we have it officially and on the website": we probably can, so long as we don't have issues with it and if we can get around to it
